src/client/config/thema.ts

```diff
@@ -107,6 +108,12 @@ export const DocumentTitles: DocumentTitlesConfig = {
     `Vergunning | ${ThemaTitles.TOERISTISCHE_VERHUUR}`,
   [AppRoutes['TOERISTISCHE_VERHUUR/VERGUNNING/LIST']]:
     `Vergunningen | ${ThemaTitles.TOERISTISCHE_VERHUUR}`,
+  [AppRoutes['VAREN']]: `${ThemaTitles.VAREN} | overzicht`,
+  [AppRoutes['VAREN/LIST']]: `Vergunningen | ${ThemaTitles.VAREN}`,
+  [AppRoutes['VAREN/DETAIL-LIGPLAATS']]: `${ThemaTitles.VAREN} | Ligplaats`,
+  [AppRoutes['VAREN/DETAIL-EXPLOITATIE']]: `${ThemaTitles.VAREN} | Exploitatie`,
+  [AppRoutes['VAREN/DETAIL-BEDRIJFSGEGEVENS']]:
+    `${ThemaTitles.VAREN} | Rederij`,
```

Comment:
Je kunt dit wat generieker noteren (icm een andere route definitie). Zo is het ook bv bij AFIS gedaan.

```suggestion
  [generatePath(AppRoutes['VAREN/DETAIL'], { caseType: 'ligplaatsvergunning' })]:
    `Ligplaatsvergunning | ${ThemaTitles.VAREN}`,
  [generatePath(AppRoutes['VAREN/DETAIL'], { caseType: 'exploitatievergunning' })]:
    `Exploitatievergunning | ${ThemaTitles.VAREN}`,
```